docs/UserGuide.md

```diff
@@ -78,18 +84,44 @@ Track the internships you're interested in and all the relevant information -- t
 
 </div>
 
+--------------------------------------------------------------------------------------------------------------------
+# 3. Quick start
+
+1. Ensure you have Java `11` or above installed in your Computer.
+
+1. Download the latest `InterNUS.jar` from [here](https://github.com/AY2223S1-CS2103T-F11-1/tp/releases).
+
+1. Copy the file to the folder you want to use as the _home folder_ for your app.
+
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+   ![Ui](images/Ui.png)
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+
+1. Refer to the [Features](#features) below for details of each command.
+
+--------------------------------------------------------------------------------------------------------------------
+
+# 4. Features
+
 ## 4.1 Add Command
 
 ### 4.1.1 Adding a person: `add -p`
 
-Adds a person to InterNUS.
+Adds a person to InterNUS. This person refers to the contact person you want to save. 
+During internship application this person could be the hiring manager. 
+After the application, this person could be the senior engineer who you want to keep contact at work.
 
 Format: `add -p n/NAME [e/EMAIL] [p/PHONE_NUMBER] [t/TAG]…​ [l/LINK_INDEX] c/[COMPANY]`
-
-<div markdown="span" class="alert alert-primary">:bulb: **![img.png](img.png)**
-A person can have any number of tags (including 0). Link index is optional.
-The link index refers to the index number shown in the internship list. Company is optional.
-The company refers to the company the contact person is working at. 
+* The link index (in add -p) refers to the index number shown in the internship list.
+* The company refers to the company the contact person is working at.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A person can have any number of tags (including 0). 
+New added person will be added in current sorted order.
+Only one contact person can be in-charge of one internship position.
+Phone number allows more than 2 digit without the need of any specific pattern.
```

Comment:
```suggestion
Phone number allows more than 2 digits without the need of any specific pattern.
```
I think this line can be moved up to the bulleted list above, as it's a constraint on a parameter and not really a "Tip".